pepper-apis/dsm-core/src/main/resources/liquibase/singular/DDP-8612_NA_Checkbox_options.xml

```diff
@@ -0,0 +1,165 @@
+            <column name="column_name" value="NA_SINGLE_VENTRICLE_GROUP"/>
+            <column name="column_display" value="N/A"/>
+            <column name="display_type" value="CHECKBOX"/>
+            <column name="order_number" value="9"/>
+        </insert>
+        <insert tableName="field_settings">
+            <column name="ddp_instance_id"
+                    valueComputed="(SELECT ddp_instance_id FROM ddp_instance WHERE instance_name='singular')"></column>
+            <column name="field_type" value="AORTOPATHY_GROUP"/>
+            <column name="column_name" value="NA_AORTOPATHY_GROUP"/>
+            <column name="column_display" value="N/A"/>
+            <column name="display_type" value="CHECKBOX"/>
+            <column name="order_number" value="13"/>
+        </insert>
+        <insert tableName="field_settings">
+            <column name="ddp_instance_id"
+                    valueComputed="(SELECT ddp_instance_id FROM ddp_instance WHERE instance_name='singular')"></column>
+            <column name="field_type" value="CORONARY_ANOMALY_GROUP"/>
+            <column name="column_name" value="NA_CORONARY_ANOMALY_GROUP"/>
+            <column name="column_display" value="N/A"/>
+            <column name="display_type" value="CHECKBOX"/>
+            <column name="order_number" value="5"/>
+        </insert>
+        <insert tableName="field_settings">
+            <column name="ddp_instance_id"
+                    valueComputed="(SELECT ddp_instance_id FROM ddp_instance WHERE instance_name='singular')"></column>
+            <column name="field_type" value="CARDIOMYOPATHY_GROUP"/>
+            <column name="column_name" value="NA_CARDIOMYOPATHY_GROUP"/>
+            <column name="column_display" value="N/A"/>
+            <column name="display_type" value="CHECKBOX"/>
+            <column name="order_number" value="7"/>
+        </insert>
+        <insert tableName="field_settings">
+            <column name="ddp_instance_id"
+                    valueComputed="(SELECT ddp_instance_id FROM ddp_instance WHERE instance_name='singular')"></column>
+            <column name="field_type" value="MYOCARDIAL_GROUP"/>
+            <column name="column_name" value="NA_MYOCARDIAL_GROUP"/>
+            <column name="column_display" value="N/A"/>
+            <column name="display_type" value="CHECKBOX"/>
+            <column name="order_number" value="13"/>
+        </insert>
+    </changeSet>
+    <changeSet id="DDP-8612_NA_checkboxes_tab3" author="pegah">
+        <sql>
+            select @newChange :=  '[{"value": "NA", "name": "N/A"}]';
+            select @pv := possible_values from field_settings where column_name = "KARYOTYPE_ABNORMAL" and field_type = 'GROUP_GENETIC_TESTING';
+            select JSON_MERGE_PRESERVE(@pv, @newChange);
+            UPDATE field_settings
+            SET possible_values =  JSON_MERGE_PRESERVE(@pv, @newChange)
+            WHERE field_settings_id &#60;&#62; 0 and field_settings_id in
```

Comment:
can we change `&#60;&#62;` to `&lt;&gt;`

Reply:
Fixed